src/RA_Dlg_AchEditor.cpp

```diff
@@ -2147,9 +2043,14 @@ void Dlg_AchievementEditor::LoadAchievement(Achievement* pCheevo, _UNUSED BOOL)
         m_bPopulatingAchievementEditorData = FALSE;
     }
 
-    //InvalidateRect( hList, nullptr, TRUE );
-    //UpdateWindow( hList );
-    //RedrawWindow( hList, nullptr, nullptr, RDW_INVALIDATE );
+    // InvalidateRect( hList, nullptr, TRUE );
+    // UpdateWindow( hList );
+    // RedrawWindow( hList, nullptr, nullptr, RDW_INVALIDATE );
+}
+
+inline char* Dlg_AchievementEditor::LbxDataAt(unsigned int nRow, CondSubItems nCol)
```

Comment:
To not have to put the enum in the header file, this is defined in the source file.